README.md

```diff
@@ -87,6 +82,10 @@ KnowSheet> HTTP(2015)
 // KnowSheet Bricks HTTPServer at port 2015
 ```
 
+* Bricks maintains a single server per port. The first call to `HTTP` with a port number starts an HTTP server on that port and returns an instance of the server. The next calls with the same port reference the started server.
```

Comment:
I like this clean phrasing. Would also add a phrase along the lines of ".. thus, the user does not have to think of starting the server itself; all that is needed to start using the endpoint is to `Register()` it".

Long, I know. Perhaps as a separate, first bullet point?